src/core/server/saved_objects/import/utils.ts

```diff
@@ -19,6 +19,43 @@ export interface UpdateDataSourceNameInVegaSpecProps {
   spacing?: number;
 }
 
+/**
+ * Given a visualization saved object and datasource id, return the updated visState and references if the visualization was a TSVB visualization
+ * @param {SavedObject} object
+ * @param {string} dataSourceId
+ * @returns {{visState: string, references: SavedObjectReference[]}} - the updated stringified visState and references
+ */
+export const getUpdatedTSVBVisState = (object: SavedObject, dataSourceId: string) => {
```

Comment:
Let's define type of object to SavedObject<T> and then object.attributes as T.

Reply:
I defined a type `VisualizationType`